types/fibjs/declare/BufferedStream.d.ts

```diff
@@ -23,116 +23,115 @@
     */
 /// <reference path="Stream.d.ts" />
 declare class Class_BufferedStream extends Class_Stream {
-    
+
     /**
-     * class prop 
+     * class prop
+     *
      *
-     * 
      * @brief 查询创建缓存对象时的流对象
-     * 
+     *
      * @readonly
-     * @type Stream
```

Comment:
There's generally no need to explicitly indicate `@type` of anything that's already given a type annotation in TypeScript. It's not generally picked up well by tooling, and is sometimes inaccurate. For example, the `Stream` type here is a human nickname for the actual `Class_Stream` type. And later on `String` is used instead of `string`.